dts/arm/st/f4/stm32f429.dtsi

```diff
@@ -58,18 +58,7 @@
 			label = "SPI_4";
 		};
 
-		i2s4: i2s@40013400 {
-			compatible = "st,stm32-i2s";
-			#address-cells = <1>;
-			#size-cells = <0>;
-			reg = <0x40013400 0x400>;
-			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00002000>;
-			interrupts = <84 5>;
-			status = "disabled";
-			label = "I2S_4";
-		};
-
-		/* spi5/i2s5 is present on all STM32F429XX SoCs except
+		/* spi5 is present on all STM32F429XX SoCs except
 		 * STM32F429vX SoCs. Delete node in stm32f429vX.dtsi.
 		 */
```

Comment:
If my understanding is correct, this SoC only has SPI2/I2S2 and SPI3/I2S3 controllers with I2S capabality.
So it seems that SPI1/SPI4/SPI5/SPI6 are only SPI. Is this correct understanding?

Reply:
@avisconti, this is corrrect